import_json.py

```diff
@@ -0,0 +1,125 @@
+import json
+import django
+from django.core.exceptions import ObjectDoesNotExist
+django.setup()
+from common.models import Journal, Organization
+
+def import_json():
+
+    print('Loading json')
+    # organization json file
+    # with open('./cos/static/toporgs.json') as data_file:
+    #     data = json.load(data_file)
+    #
+    #     for i in data:
+    #         name = i['Organization']
+    #         try:
+    #             x = Organization.objects.get(name=name)
+    #         except ObjectDoesNotExist:
+    #             x = Organization.objects.create(name=name)
+    #         x.save()
+
+    print('Finished loading toporgs.json')
+    print('Start loading journal json files')
+    # journal first json file
+    with open('./cos/static/rrjournals.json') as data_file:
+        data = json.load(data_file)
+
+        for i in data:
+            title = i['Title']
+
+            try:
+                x = Journal.objects.get(title=title)
+            except ObjectDoesNotExist:
+                x = Journal.objects.create(title = title)
+                x.url_link = i['URL']
+                notes = []
+                for note in i['Notes']:
+                    notes.append(('note', {'description': note['Description'], 'link': note['Link']}))
+                x.notes = notes
+            x.is_registered_journal = True
+            x.save()
+
+    # second json file
+    with open('./cos/static/rrjournalssome.json') as data_file:
```

Comment:
I was thinking about creating a function combining the functionality. I will update that later.